coherence/upnp/devices/media_server.py

```diff
@@ -385,7 +385,12 @@ def process_child(self, ch, name, request):
                     self.warning(
                         f'{request.method} request with content-length '
                         f'{headers[b"content-length"]} header - sanitizing')
-                    del request.received_headers[b'content-length']
+					# This should fix #20
+                    try:
+                        del request.received_headers[b'content-length']
+                    except:
```

Comment:
Since we know that the failure come from a missing attribute from requests, I would  specify the exception (which also will fix the [remaining linter error](https://travis-ci.org/opacam/Cohen3/jobs/571920850#L1467)):
```
                    except AttributeError:
```